src/assets/bundles/bundle_ru.properties

```diff
@@ -5,22 +5,22 @@
 #=#=#=#=#=#=#=#=#=#=#=#=#=# IDK TITLE #=#=#=#=#=#=#=#=#=#=#=#=#=#
 
 #=#=# UI #=#=#
-setting.aeyama-checkUpdate.name = Check for Updates
-setting.aeyama-checkUpdate.description = Choose if the mod checks for updates.
-setting.aeyama-checkBetaUpdate.name = Check for Beta Updates
-setting.aeyama-checkBetaUpdate.description = Choose if the mod checks for beta updates.\n[red]Need the "Check for Updates" setting enabled.
-
-stat.armors = Armors
-stat.weapons = Weapons
-
-aeyama.updater.name = [scarlet]\u26A0[] Outdated Mod!
-aeyama.updater.info = Your version of [accent]Aeyama[] is outdated.\n\n[gray]Installed version: [lightgray]v{0}[]\nLatest version: [lightgray]v{1}[].[]
-aeyama.updater.info-beta = Your version of [accent]Aeyama[] is outdated.\n\n[gray]Installed version: [lightgray]v{0}[]\nLatest version: [lightgray]v{1}[]\nBeta version: [lightgray]v{2}[].[]
-aeyama.updater.install = Install Latest
-aeyama.updater.install-beta = Install Beta
-aeyama.updater.ignore = Ignore
-
-aeyama.news.title = Thanks for installing Aeyama: The New World!\nInstalled version: v{0}
+setting.aeyama-checkUpdate.name = Проверять обновления
+setting.aeyama-checkUpdate.description = Выберите, будет ли мод проверять обновления
+setting.aeyama-checkBetaUpdate.name = Проверять бета релизы
+setting.aeyama-checkBetaUpdate.description = Выберите, будет ли мод проверять бета релизы.\n[red]Требуется включить функцию "Проверять обновления"!
+
+stat.armors = Броня
+stat.weapons = Оружие
+
+aeyama.updater.name = [scarlet]\u26A0[] Устаревший мод!
+aeyama.updater.info = Ваша версия мода устарела.\n\n[gray]Установленная версия: [lightgray]v{0}[]\nПоследняя версия: [lightgray]v{1}\n[white]Выберите версию для установки:[][][]
+aeyama.updater.info-beta = Ваша версия мода устарела.\n\n[gray]Установленная версия: [lightgray]v{0}[]\nПоследняя версия: [lightgray]v{1}[]\nБета релиз: [lightgray]v{2}\n[white]Выберите версию для установки: [][][]
```

Comment:
Why did you remove the `[accent]`?

Reply:
I removed the mod's name because it felt like the game wanted to update this mod instead of mod to update itself. I can revert this